prow/pod-utils/downwardapi/jobspec.go

```diff
@@ -95,6 +97,20 @@ const (
 	pullPullShaEnv = "PULL_PULL_SHA"
 )
 
+// EnvForSpec returns a list of EnvVars
+// to their values that should be available for a job spec
+func KubeEnv(spec JobSpec) ([]coreapi.EnvVar, error) {
```

Comment:
Sorry -- I meant take the `kubeEnv` function in `decorate/podspec.go` and expose it

Reply:
Done.
Sorry, not thinking straight now.